src/cts/src/HTreeBuilder.cpp

```diff
@@ -224,6 +225,456 @@ void HTreeBuilder::initSinkRegion()
   logger_->info(CTS, 26, "    Height: {:.4f}.", sinkRegion_.getHeight());
 }
 
+// xxx: plot blockages
+void plotBlockage(std::ofstream& file, odb::dbDatabase* db_, int z)
+{
+  unsigned i = 0;
+  for (odb::dbBlockage* blockage : db_->getChip()->getBlock()->getBlockages()) {
+    // std::string name = inst->getName();
+    std::string name = "xxx";
```

Comment:
not a helpful name - why is the first line commented out?

Reply:
Fixed